web/packages/teleport/src/Welcome/NewCredentials/NewMfaDevice.tsx

```diff
@@ -144,94 +144,95 @@ export function NewMfaDevice(props: NewMfaDeviceProps) {
           <Flex
             flexDirection="column"
             justifyContent="center"
-            alignItems="center"
             borderRadius={8}
             bg={mfaType?.value === 'optional' ? 'levels.elevated' : ''}
-            height={mfaType?.value === 'optional' ? '340px' : '240px'}
-            px={3}
           >
+            {(mfaType?.value === 'otp' ||
+              (mfaType?.value === 'webauthn' && !!credential)) && (
+              <FieldInput
+                rule={requiredField('MFA method name is required')}
+                label="MFA method name"
+                placeholder="Name"
+                ref={deviceNameInputRef}
+                value={deviceName}
+                type="text"
+                onChange={e => setDeviceName(e.target.value)}
+                readonly={submitAttempt.status === 'processing'}
+                mb={3}
+              />
+            )}
             {mfaType?.value === 'otp' && (
-              <>
+              <Flex
+                mb={4}
+                border={1}
+                borderColor="interactive.tonal.neutral.2"
+                borderRadius={3}
+                p={3}
+                gap={3}
+              >
                 <Image
-                  src={imgSrc}
-                  width="145px"
-                  height="145px"
+                  src={qrCodeImage}
+                  width="168px"
+                  height="168px"
                   css={`
                     border: 4px solid white;
+                    box-sizing: border-box;
+                    border: 8px solid white;
+                    border-radius: 8px;
                   `}
                 />
-                <Text
-                  fontSize={1}
-                  textAlign="center"
-                  mt={2}
-                  color="text.slightlyMuted"
-                >
-                  Scan the QR Code with any authenticator app and enter the
-                  generated code. We recommend{' '}
-                  <Link href="https://authy.com/download/" target="_blank">
```

Comment:
Strange, I could swear you already removed this recently. 🤷

Reply:
I did, on a different component :)